cmd/commands/runtime.go

```diff
@@ -292,6 +307,172 @@ func createRuntimeOnPlatform(ctx context.Context, opts *model.RuntimeInstallatio
 	return runtimeCreationResponse.NewAccessToken, nil
 }
 
+func ensureClusterRequirements(ctx context.Context, kubeFactory kube.Factory, namespace string) error {
+	requirementsValidationErrorMessage := "cluster does not meet minimum requirements"
+	var specificErrorMessages []string
+	
+	client, err := kubeFactory.KubernetesClientSet()
+	if err != nil {
+		return fmt.Errorf("%s: cannot create kubernetes clientset: %v ", requirementsValidationErrorMessage, err)
```

Comment:
no need for the `cluster does not meet minimum requirements` here. maybe just `failed to verify cluster requirements: cannot create kubernetes clientset: %w`...